lib/evmone/instructions.hpp

```diff
@@ -1217,6 +1436,7 @@ inline TermResult selfdestruct(StackTop stack, int64_t gas_left, ExecutionState&
         if (state.rev < EVMC_LONDON)
             state.gas_refund += 24000;
     }
+    // TODO any symbolic requirements??
```

Comment:
I believe we need exact match on beneficiary. At least exact match on `state.host.access_account(beneficiary) == EVMC_ACCESS_COLD` and `state.host.account_exists(beneficiary)` in some cases.